pages/Messages/Announce.md

```diff
@@ -30,29 +30,29 @@ menu: Messages
 The DSNPBatch event is a standard event topic to be used for announcing a location that a list of [DSNP Messages](/DSNP/DSNP-Messages) can be found.
 
 ```solidity
-interface DSNPAnnounce {
-    event DSNPBatch(bytes32 dsnpHash, bytes[] dsnpUri);
+interface IAnnounce {
```

Comment:
@enddynayn IAnnounce is getting caught by our spellchecker. https://github.com/LibertyDSNP/spec/blob/main/.spelling